lib/fizzy/execute.cpp

```diff
@@ -837,10 +837,10 @@ ExecutionResult execute(Instance& instance, FuncIdx func_idx, const Value* args,
             try
             {
                 if (new_pages > instance.memory_pages_limit)
-                    throw std::bad_alloc();
+                    throw std::length_error("");
```

Comment:
Even though we do not use `bad_alloc` here, the underlying resize can still emit it according to the docs.

Reply:
Yes, it's better to add it back, but it will be difficult to cover it with tests.